sums.rb

```diff
@@ -0,0 +1,25 @@
+class Sum1
+    attr_accessor :total
+  
+    def initialize(a, b)
+      @total = a + b
+    end
+  end
+  
+  class Sum2
+    attr_reader :a, :b
```

Comment:
Because you only call `new_total` without attempting to directly access `a` or `b` from outside of the class, you don't need the `attr_reader`.